src/renate/utils/deepspeed.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 The PyTorch Lightning team and Microsoft Corporation. All rights reserved.
```

Comment:
this file in particular seems to need a bit more coverage

Reply:
Unsure how.